src/index.html

```diff
@@ -3,10 +3,391 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="https://www.moyo.ua/ua"
+      >
+        <img
+          class="logo__img"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a class="nav__link is-active" href="https://www.moyo.ua/ua/telecommunication/smart/apple/">
```

Comment:
It will be better, if you dont use separate `class "is-active"`, you can do it with BEM-mixes, for example:
`class="nav__link nav__link-active"`)

Reply:
This changes could be optional, if you think that you wrote everything correct, then sent it to mentor review)
Good job!)🔥

Reply:
Yes, I could but it was one of the conditions of the task to add the class "is-active". Anyway, thank you ;)